README.md

```diff
@@ -47,8 +47,9 @@ To add new tests, edit the `*_spec.js` files in `/spec/javascripts/`.
 
 ## Options
 
-| Methods | Use | Usage (Example) |
-|---------|-----|-----------------|
-|getLat   | Used to get the current latitude of the center of the map. | blurredLocation.getLat() //This would return the value in numerics |
-|getLon   | Used to get the current latitude of the center of the map. | blurredLocation.getLon() //This would return the value in numerics |
-|goTo     | Takes in three parameters, namely latitude, longitude and zoom. Will set the center of map to co-ordinates input.| blurredLocation.goTo(44.51, -89.99, 13) //Will set center of map to (44.51,-89.99) with zoom set as 13 |
+| Methods | Use | Usage (Example)|
+|---|---|---|---|---|
+|getLat   |Used to get the current latitude of the center of the map.|  blurredLocation.getLat() //This would return the value in numerics|
+|getLon|Used to get the current latitude of the center of the map|blurredLocation.getLon() //This would return the value in numerics|
+|goTo   |Takes in three parameters, namely latitude, longitude and zoom. Will set the center of map to co-ordinates input.|blurredLocation.goTo(44.51, -89.99, 13) //Will set center of map to (44.51,-89.99) with zoom set as 13|
+|getSize   |Used to get the current size of the map, namely the width and height.| blurredLocation.getSize() //This would return the size of the map|
```

Comment:
This looks good, and I assume it's needed for some of the other functions. But let's keep in mind what is essential API that cannot be provided by direct access to the map, such as a native Leaflet call (not sure but probably `map.getSize()`?) Can this be done here without a redundant wrapper, or is `getSize` really necessary for the API? Thanks!